Resources/Prototypes/Loadouts/loadout_groups.yml

```
@@ -1172,6 +1172,10 @@
   loadouts:
   - ResearchDirectorJumpsuit
   - ResearchDirectorJumpskirt
+  - ResearchDirectorJumpsuitShirt
+  - ResearchDirectorJumpskirtShirt
+  - ResearchDirectorJumpsuitSuit
+  - ResearchDirectorJumpskirtSuit
```

```suggestion
  - ResearchDirectorJumpsuitShirt # DS14
  - ResearchDirectorJumpskirtShirt # DS14
  - ResearchDirectorJumpsuitSuit # DS14
  - ResearchDirectorJumpskirtSuit # DS14
```

И так везде где изменяешь прототипы визардов